### ISSUE TITLE
QGrid is not working with NB-viewer.

### ISSUE THREAD

**Issue by muneebable** [quantopian/qgrid#381]
### Environment

* Operating System: Ubuntu 
* Python Version: `3.6.9`
* How did you install Qgrid: `pip`
* Python packages: absl-py==0.13.0
aiobotocore==1.4.2
aiohttp==3.7.4.post0
aiohttp-cors==0.7.0
aiohttp-retry==2.4.6
aioitertools==0.10.0
alembic==1.6.5
altair==4.1.0
altgraph==0.17
appdirs==1.4.4
argon2-cffi==20.1.0
astor==0.8.1
async-generator==1.10
async-timeout==3.0.1
atpublic==2.3
attrs==21.2.0
backcall==0.2.0
backports.zoneinfo==0.2.1
base58==2.1.0
basemap==1.3.6
basemap-data==1.3.2
beautifulsoup4==4.9.3
BentoML==0.13.1
bitarray==1.2.2
black==22.8.0
bleach==3.3.1
blinker==1.5
boto3==1.17.106
botocore==1.20.106
Bottleneck==1.3.2
branca==0.4.2
bravado==11.0.3
bravado-core==5.17.0
Brotli==1.0.9
cached-property==1.5.2
cachetools==4.2.4
Cerberus==1.3.4
certifi==2021.5.30
cffi==1.14.6
chardet==4.0.0
charset-normalizer==2.0.3
click==7.1.2
click-plugins==1.1.1
cligj==0.7.2
colorama==0.4.4
comet-ml==3.28.0
commonmark==0.9.1
configobj==5.0.6
configparser==5.0.2
contextlib2==21.6.0
contextvars==2.4
cycler==0.10.0
cytoolz==0.11.0
dash==2.0.0
dash-bootstrap-components==1.2.1
dash-colorscales==0.0.4
dash-core-components==2.0.0
dash-daq==0.5.0
dash-html-components==2.0.0
dash-table==5.0.0
dataclasses==0.8
dataframe-image==0.1.1
decorator==4.4.2
deepmerge==0.3.0
defusedxml==0.7.1
dictdiffer==0.9.0
diskcache==5.4.0
distro==1.7.0
docker==5.0.0
docutils==0.17.1
dpath==2.0.6
dtale==2.9.1
dulwich==0.20.33
dvc==2.8.1
elasticsearch==7.13.4
entrypoints==0.3
et-xmlfile==1.1.0
eth-abi==2.1.1
eth-account==0.5.6
eth-hash==0.3.2
eth-keyfile==0.5.1
eth-keys==0.3.3
eth-rlp==0.2.1
eth-typing==2.2.2
eth-utils==1.10.0
everett==3.0.0
feather-format==0.4.1
Fiona==1.8.20
Flask==1.1.4
Flask-Compress==1.13
flask-ngrok==0.0.25
flatten-dict==0.4.2
flufl.lock==3.2
folium==0.12.1
fsspec==2021.10.0
ftfy==6.0.3
funcy==1.17
future==0.18.2
gast==0.5.0
Geohash==1.0
geopandas==0.9.0
gitdb==4.0.9
GitPython==3.1.18
grandalf==0.6
grpcio==1.39.0
gunicorn==20.1.0
h5py==3.1.0
hexbytes==0.2.2
htmlmin==0.1.12
humanfriendly==9.2
idna==3.2
idna-ssl==1.1.0
ImageHash==4.2.1
immutables==0.15
importlib-metadata==4.8.3
importlib-resources==5.2.3
ipfshttpclient==0.8.0a2
ipykernel==5.5.5
ipython==7.16.1
ipython-genutils==0.2.0
ipywidgets==7.6.3
itsdangerous==1.1.0
jedi==0.18.0
Jinja2==2.11.3
jmespath==0.10.0
joblib==1.0.1
jsonpointer==2.2
jsonref==0.2
jsonschema==3.2.0
jupyter==1.0.0
jupyter-client==6.1.12
jupyter-console==6.4.0
jupyter-contrib-core==0.3.3
jupyter-contrib-nbextensions==0.5.1
jupyter-core==4.7.1
jupyter-highlight-selected-word==0.2.0
jupyter-latex-envs==1.4.6
jupyter-nbextensions-configurator==0.4.1
jupyterlab-pygments==0.1.2
jupyterlab-widgets==1.0.0
kaleido==0.1.0
Keras==2.2.5
Keras-Applications==1.0.8
Keras-Preprocessing==1.1.2
kiwisolver==1.1.0
lru-dict==1.1.7
lxml==4.8.0
lz4==3.1.10
mailchecker==4.1.17
Mako==1.1.4
mapclassify==2.4.3
Markdown==3.3.4
MarkupSafe==2.0.1
matplotlib==3.3.4
missingno==0.4.2
missingpy==0.2.0
mistune==0.8.4
mmh3==3.0.0
mock==4.0.3
monotonic==1.6
msgpack==1.0.3
multiaddr==0.0.9
multidict==5.1.0
multimethod==1.4
munch==2.5.0
mypy-extensions==0.4.3
nanotime==0.5.2
nbclient==0.5.3
nbconvert==6.0.7
nbformat==5.1.3
neptune-client==0.15.1
neptune-tensorflow-keras==0.9.9
nest-asyncio==1.5.1
netaddr==0.8.0
networkx==2.5.1
notebook==6.4.0
numpy==1.19.5
nvidia-ml-py3==7.352.0
oauthlib==3.2.0
opencv-python==4.5.5.64
openpyxl==3.0.8
packaging==21.0
pandas==1.1.5
pandocfilters==1.4.3
parsimonious==0.8.1
parso==0.8.2
pathspec==0.9.0
patsy==0.5.1
pep517==0.11.0
pexpect==4.8.0
phik==0.12.0
phonenumbers==8.12.49
pickleshare==0.7.5
Pillow==8.4.0
pip-compile-multi==2.4.1
pip-tools==6.2.0
platformdirs==2.4.0
plotly==5.1.0
ply==3.11
prometheus-client==0.11.0
prompt-toolkit==3.0.19
protobuf==3.17.3
psutil==5.8.0
ptyprocess==0.7.0
pyarrow==4.0.1
pyasn1==0.4.8
pycparser==2.20
pycryptodome==3.11.0
pydantic==1.8.2
pydeck==0.6.2
pydot==1.4.2
pyee==8.2.2
pygit2==1.6.1
Pygments==2.9.0
pygtrie==2.4.2
pyinstaller==4.4
pyinstaller-hooks-contrib==2021.2
PyJWT==2.3.0
Pympler==1.0.1
pyparsing==2.4.7
pyppeteer==0.2.6
pyproj==3.0.1
pyrsistent==0.18.0
pyshp==2.3.1
python-benedict==0.25.1
python-dateutil==2.8.2
python-editor==1.0.4
python-fsutil==0.6.1
python-geohash==0.8.5
python-json-logger==2.0.1
python-slugify==6.1.2
pytz==2021.1
pytz-deprecation-shim==0.1.0.post0
PyWavelets==1.1.1
PyYAML==6.0
pyzmq==22.1.0
qgrid==1.3.1
qtconsole==5.1.1
QtPy==1.9.0
requests==2.26.0
requests-oauthlib==1.3.1
requests-toolbelt==0.9.1
rfc3987==1.3.8
rich==12.4.4
rlp==2.0.1
ruamel.yaml==0.17.21
ruamel.yaml.clib==0.2.6
s3fs==2021.10.0
s3transfer==0.4.2
schema==0.7.4
scikit-learn==0.24.2
scipy==1.5.4
seaborn==0.11.2
selenium==3.141.0
semantic-version==2.9.0
semver==2.13.0
Send2Trash==1.7.1
Shapely==1.7.1
shortuuid==1.0.9
shtab==1.5.4
simple-di==0.1.0
simplejson==3.17.6
six==1.16.0
smmap==5.0.0
soupsieve==2.2.1
SQLAlchemy==1.3.24
SQLAlchemy-Utils==0.36.5
squarify==0.4.3
statsmodels==0.12.2
streamlit==1.10.0
strict-rfc3339==0.7
strsimpy==0.2.1
swagger-spec-validator==2.7.4
tabulate==0.8.9
tangled-up-in-unicode==0.1.0
tenacity==8.0.1
tensorboard==1.13.1
tensorflow==1.13.2
tensorflow-estimator==1.13.0
termcolor==1.1.0
terminado==0.10.1
testpath==0.5.0
text-unidecode==1.3
threadpoolctl==2.2.0
toml==0.10.2
tomli==1.2.3
toolz==0.11.1
toposort==1.6
tornado==6.1
tqdm==4.62.2
traitlets==4.3.3
typed-ast==1.5.4
types-dataclasses==0.1.5
typing-extensions==4.1.1
tzdata==2022.2
tzlocal==4.2
urllib3==1.25.11
validators==0.20.0
varint==1.0.2
visions==0.7.1
voluptuous==0.13.1
watchdog==2.1.9
wcwidth==0.2.5
web3==5.24.0
webcolors==1.11.1
webencodings==0.5.1
websocket-client==1.1.0
websockets==9.1
Werkzeug==1.0.1
widgetsnbextension==3.5.1
wrapt==1.14.0
wurlitzer==3.0.2
xarray==0.16.2
xlrd==2.0.1
xmltodict==0.13.0
yarl==1.6.3
zc.lockfile==2.0
zipp==3.6.0

* Jupyter lab packages (if applicable): `$ jupyter labextension list`

### Description of Issue

* What did you expect to happen?
* What happened instead?

### Reproduction Steps

1.The qgrid is working fine on JUpyter notebook locally. I have a nb-viewer server, when I upload it there, it doesnot show me the table for which I use qgrid.
...

### What steps have you taken to resolve this already?
- tried to use older version but doesnot solve my problem.
- trying to find a js file to be locally embedded in notebook.
...

### Anything else?
![image](https://user-images.githubusercontent.com/3126387/211582083-9a40b800-6b82-4362-94ca-bf4c79955f9e.png)

...
